substrate/frame/support/procedural/src/pallet/expand/tt_default_parts.rs

```diff
@@ -138,5 +192,25 @@ pub fn expand_tt_default_parts(def: &mut Def) -> proc_macro2::TokenStream {
 		}
 
 		pub use #extra_parts_unique_id as tt_extra_parts;
+
+		#[macro_export]
+		#[doc(hidden)]
+		macro_rules! #default_parts_unique_id_v2 {
+			{
+				$caller:tt
+				frame_support = [{ $($frame_support:ident)::* }]
```

Comment:
Is this passing in frame-support in case its renamed or what is going on?

Reply:
Exactly - its passed in as here: https://github.com/paritytech/polkadot-sdk/blob/a5401ed8e258b0a3d00318138987985db75924b0/substrate/frame/support/procedural/src/runtime/expand/mod.rs#L91